### DISCUSSION TITLE
Error: Mismatched LaTeX support files detected. \ExplLoaderFileDate{expl3.sty}}

### DISCUSSION THREAD

**Discussion by earthmoon** [whutug/whu-thesis#180] [Q&A] [answered]
你好，
今天更新了 Tex Live manager, 然后运行后报错：
`Mismatched LaTeX support files detected. \ExplLoaderFileDate{expl3.sty}}`
![image](https://user-images.githubusercontent.com/45897471/201467279-a66234a2-cfbb-4f60-8544-32d7515e116f.png)
这会是什么问题呢？
谢谢！

**Comment by sikouhjw** [comment, marked as answer]
命令行执行
```
fmtutil-sys --all
```

**Reply by earthmoon** [reply]
谢谢。这个办法可以解决这个问题。
